_posts/2022-11-25-prac1b-data-annotation.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+---
+
+* This will become a table of contents (this text will be scrapped).
```

Comment:
The TOC for 1b seems to be messed up with the numbers. Might need to double check the headings.

![Screenshot 2022-11-25 at 16 20 57](https://user-images.githubusercontent.com/7655454/204024405-f3c08d0e-976b-400a-901c-c1e6a1012b64.png)